utils/generator.py

```diff
@@ -258,7 +309,9 @@ def add_attribute(self, k, arg, for_types_py=False):
             type_ = "Any"
             param = None
         if type_ != "Any":
-            if "types." in type_:
+            if 'Sequence["types.' in type_:
```

Comment:
This is not a great way to do this. I'm thinking it may be nice to have a library that understands Python type hints and is able to parse them and render them, but for now this finds all the sequences.